user/docker.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Using Docker in Builds
+layout: en
+permalink: /user/docker/
+---
+
+<div id="toc"></div>
+
+Travis CI builds can run and build Docker images, and can also push images to
+Docker repositories or other remote storage.
+
+<!-- @joshk -->
+
+If you don't already have access to the beta workers running Ubuntu Trusty
+Tahr, please [contact support](mailto:support@travis-ci.com) and let them know
+which repository you'd like to use Docker in.
+
+To use Docker you need the following settings in your `.travis.yml`:
+
+```
+sudo: required
+
+dist: trusty
```

Comment:
The simplest way now to get running on the setup is to add this line:

```yaml
services:
  - docker
```

The two lines above will also get you on this setup, but this is a more expressive way of telling us that you want to run on our Docker-able stack.